src/components/Calculator.js

```diff
@@ -1,31 +1,43 @@
 /* eslint-disable react/prefer-stateless-function */
 import React from 'react';
+import calculate from '../logic/calculate';
 import './Calculator.css';
 
 class Calculator extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {};
+    this.handleClick = this.handleClick.bind(this);
+  }
+
+  handleClick(e) {
+    this.setState((state) => calculate(state, e.target.textContent));
+  }
+
   render() {
+    const { next, total } = this.state;
     return (
       <div className="calc-container">
-        <div className="output bg-grey">0.0</div>
-        <button type="button" className="bg-onix">AC</button>
-        <button type="button" className="bg-onix">+/-</button>
-        <button type="button" className="bg-onix">%</button>
-        <button type="button" className="bg-orange">÷</button>
-        <button type="button" className="bg-onix">7</button>
-        <button type="button" className="bg-onix">8</button>
-        <button type="button" className="bg-onix">9</button>
-        <button type="button" className="bg-orange">x</button>
-        <button type="button" className="bg-onix">4</button>
-        <button type="button" className="bg-onix">5</button>
-        <button type="button" className="bg-onix">6</button>
-        <button type="button" className="bg-orange">-</button>
-        <button type="button" className="bg-onix">1</button>
-        <button type="button" className="bg-onix">2</button>
-        <button type="button" className="bg-onix">3</button>
-        <button type="button" className="bg-orange">+</button>
-        <button type="button" className="span-two">0</button>
-        <button type="button" className="bg-onix">.</button>
-        <button type="button" className="bg-orange">=</button>
+        <div className="output bg-grey">{next || total || 0}</div>
+        <button type="button" className="bg-onix" onClick={this.handleClick}>AC</button>
+        <button type="button" className="bg-onix" onClick={this.handleClick}>+/-</button>
+        <button type="button" className="bg-onix" onClick={this.handleClick}>%</button>
```

Comment:
- [ ] [OPTIONAL]One of the beauties of React is reusable components. I'd suggest you create a reusable `<Button />` component in order to make your project even better 👍 
- [ ] [OPTIONAL] It would be nice if the cursor becomes a pointer when a user hovers over any of the calculator buttons. This one will improve the user experience of your app.